pkg/dependencymagent/doc.go

```diff
@@ -0,0 +1,11 @@
+// +build tools
```

Comment:
any reason already existing setup for tools doesn't work here https://github.com/openshift/installer/blob/master/tools.go

Reply:
Nope. I just did not realize it existed. I will move to that. Thanks.